.gitignore

```diff
@@ -9,6 +9,8 @@
 *.save
 *.autosav
 *.autosave
+*.swp
```

Comment:
Created by vim https://stackoverflow.com/questions/821902/disabling-swap-files-creation-in-vim